pages/osx/whence.md

```diff
@@ -0,0 +1,19 @@
+# whence
+
+> A zsh builtin to indicate how a given command would be interpreted.
+
+- Interpret {{command}}, with expansion if defined as an `alias` (similar to `command -v` builtin):
+
+`whence {{command}}`
+
+- Display type of {{command}}, with location if defined as a function, or binary (equivalent to `type` and `command -V` builtins):
+
+`whence -v {{command}}`
+
+- Same as above, except display content of shell functions instead of location (equivalent to `which` builtin):
```

Comment:
The others are builtins, but I don't think `which` is (try `which which` and see for yourself). Perhaps we could update this - what about this?
```suggestion
- Same as above, except display content of shell functions instead of location (equivalent to the `which` command):
```

Reply:
`which` is a shell builtin, as well as `/usr/bin/which`. If you're on macOS like me, you can do `builtin whence -va which` and see for yourself @sbrl…

Reply:
@sbrl I've clarified this in another entry; please see latest changes…